webapp/apifuncs.py

```diff
@@ -202,6 +202,31 @@ def add_todays_results(result1, result2, playercode, fullname, pos, clubname):
     return result
 
 
```

Comment:
Good question, in effect we're defining a global variable; if we're being less casual about it we should put it up at the header of the file and capitalize its name so that it doesn't get lost among functions and so that future generations understand that it's a global variable.  To your question, it's legal to define variables or perform arithmetic or whatever deep in a file between functions but it's not especially good style IMO.  If you want me to fix now I'll do so else it'll wait for you to add more translations?